foundry/xtract_method.py

```diff
@@ -135,7 +135,7 @@ def download_file(file):
         return {file["path"] + " status": True}
 
     results = Parallel(n_jobs=num_cores)(
-        delayed(download_file)(file) for file in file_ls
```

Comment:
This seems wrong: the `delayed` wrapper in joblib is what makes it execute the code in parallel. As written, this will execute the function and then submit it to be executed rather than run things asynchronously.

Reply:
I'm not sure, all I changed there was I switched to the decorator syntactic sugar so the line is easier to read

Reply:
🤦I looked for and, somehow, missed that you did use the decorator.

Nothing to see here! Carry on!